extensions/sql-database-projects/src/projectProvider/projectProvider.ts

```diff
@@ -74,4 +74,54 @@ export class SqlDatabaseProjectProvider implements dataworkspace.IProjectProvide
 
 		return vscode.Uri.file(projectFile);
 	}
+
+	/**
+	 * Gets the supported project types
+	 */
+	getProjectToolbarActions(): dataworkspace.IProjectAction[] {
+		return [{
+			id: constants.addItemActionTypeId,
+			displayName: constants.addItemActionTypeDisplayName,
+			icon: IconPathHelper.add
+		},
+		{
+			id: constants.schemaCompareActionTypeId,
+			displayName: constants.schemaCompareActionTypeDisplayName,
+			icon: IconPathHelper.schemaCompare
+		},
+		{
+			id: constants.buildActionTypeId,
+			displayName: constants.buildActionTypeDisplayName,
+			icon: IconPathHelper.build
+		},
+		{
+			id: constants.publishActionTypeId,
+			displayName: constants.publishActionTypeDisplayName,
+			icon: IconPathHelper.publish
+		},
+		{
+			id: constants.targetPlatformActionTypeId,
```

Comment:
this looks really weird to me in screenshot, for consistency, please get an icon for this action, also change the name to make it an action, e.g. Change target platform

Reply:
Added an icon here (received from Raden). Updated the screenshot as well.